app/views/posts/index.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <main class="top-offset">
   <% unless params[:page] %>
-    <h2 class="mt-1/2 <%= "text-black" unless current_user.present? && current_user.simple_view? %>">
+    <h2 class="mt-1/2">
```

Comment:
Removing the `text-black` CSS class causes the On Fire header to fail the WCAG contrast test.
![image](https://user-images.githubusercontent.com/10406383/185458845-b007b3cc-0d22-4e8c-9921-d939687a13ef.png)

```suggestion
    <h2 class="mt-1/2 text-black">
```
![image](https://user-images.githubusercontent.com/10406383/185458945-2814f023-c25f-4b01-90bf-9cc564b88007.png)

Reply:
Ah I didn't intend to remove that anyway, I read the if statement backwards!